### ISSUE TITLE
bioconda installable

### ISSUE THREAD

**Issue by SilasK** [dutilh/CAT#11]
Hey @bastiaanvonmeijenfeldt 

I' would make a bioconda installable of CAT.
Could you give CAT a license?

Do you have any test to run?

If I'm not wrong you can make the `CAT` executable. This information is somehow stored even on github. 

It would be great if you could update the version and sha256 when you have new releases.

**Comment by bastiaanvonmeijenfeldt** [comment]
Hi @SilasK,

Thanks for the feedback! All of your suggestions have been added to the to-do list, and will be included in future releases!

Best wishes,

Bastiaan

**Comment by SilasK** [comment]
Do you know which license you want to give CAT? In theory, I'm not allowed to use your code without a license.

**Comment by SilasK** [comment]
status: https://github.com/bioconda/bioconda-recipes/pull/12536

**Comment by SilasK** [comment]
Yes It needs a license.
@bastiaanvonmeijenfeldt You can modify the recipe [here](https://github.com/SilasK/bioconda-recipes/tree/603147a78e8c3671fa754316c537a4cd77c0ba8b/recipes/cat)

**Comment by bastiaanvonmeijenfeldt** [comment]
Hi Silas,

CAT will be available under CC BY 2.0. Thanks for your work on the bioconda recipe!

**Comment by SilasK** [comment]
It should bwork now!
`conda install -c bioconda cat`

**Comment by bastiaanvonmeijenfeldt** [comment]
It's not working for me yet:

```
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - cat

Current channels:

  - https://conda.anaconda.org/bioconda/linux-64
  - https://conda.anaconda.org/bioconda/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/free/linux-64
  - https://repo.anaconda.com/pkgs/free/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/pro/linux-64
  - https://repo.anaconda.com/pkgs/pro/noarch

```

Also, I'm looking into the licenses and it will be CC BY 3.0.

**Comment by bastiaanvonmeijenfeldt** [comment]
Great, thanks! There is a preprint but it's super old and CAT has changed a lot since then so not worth mentioning...

**Comment by SilasK** [comment]
@bastiaanvonmeijenfeldt Where are the releases gone? The bioconda package needs downloading it from the github.  or any other stable url..

**Comment by bastiaanvonmeijenfeldt** [comment]
Yes sorry for that! We have moved stuff around since yesterday and in the process removed the released. A new one will come online shortly!

**Comment by SilasK** [comment]
@bastiaanvonmeijenfeldt 

The cond build didn't worked because 
psutil
is an additonal requeirement. I fixed it now. any other package to install?

**Comment by bastiaanvonmeijenfeldt** [comment]
Thanks for pointing this out to me! I've included psutil to get memory size but I don't want to have another dependency. All our other python imports are standard library. So I will remove the psutil import soon and fix this in another way...

List of packages CAT/BAT uses:

argparse
datetime
gzip
math
multiprocessing
os
**psutil**
subprocess
sys
tarfile
urllib.request

**Comment by SilasK** [comment]
For the condo package there is no need to drop plutil.

**Comment by bastiaanvonmeijenfeldt** [comment]
I have removed the psutil dependency with the latest release (4.3.1). I'd rather have a dependency less for people that are only running a bare version of Python.

**Comment by SilasK** [comment]
I could install CAT also on osx. Do you think it could work?

**Comment by bastiaanvonmeijenfeldt** [comment]
There is actually no reason it should not work, *EXCEPT* for the psutil-free memory check I just built in, haha. But this is not a big deal, I can make that work on mac OS as well, working on that now!

That being said, we have extensively tested CAT/BAT on Linux, and have not on mac OS, so no guarantees here. But I will build in mac compatibility and let's see what users think. :)

**Comment by bastiaanvonmeijenfeldt** [comment]
I have made CAT/BAT Mac OS compatible, the latest version is online.

@SilasK: I think it would be easiest if I update the sha256 and version in the Bioconda recipe every time I publish a new version, where can I do that? Is simply pushing the change enough or should there be versioning/tagging/releasing?
